vmc/resource_vmc_sddc.go

```diff
@@ -441,6 +481,28 @@ func resourceSddcUpdate(d *schema.ResourceData, m interface{}) error {
 		}
 		d.Set("sddc_name", sddc.Name)
 	}
+
+	if d.HasChange("edrs_policy_type") || d.HasChange("enable_edrs") || d.HasChange("min_hosts") || d.HasChange("max_hosts") {
+		edrsPolicyClient := autoscalercluster.NewDefaultEdrsPolicyClient(connector)
+		clusterID := d.Get("cluster_id").(string)
+		minHosts := int64(d.Get("min_hosts").(int))
+		maxHosts := int64(d.Get("max_hosts").(int))
+		policyType := d.Get("edrs_policy_type").(string)
+		enableEDRS := d.Get("enable_edrs").(bool)
+		edrsPolicy := &autoscalermodel.EdrsPolicy{
+			EnableEdrs: enableEDRS,
+			PolicyType: &policyType,
+			MinHosts:   &minHosts,
+			MaxHosts:   &maxHosts,
+		}
+		_, err := edrsPolicyClient.Post(orgID, sddcID, clusterID, *edrsPolicy)
+		if err != nil {
+			return HandleUpdateError("EDRS Policy", err)
+		}
+
+		// To be removed once the API spec has been fixed to return the task ID
+		time.Sleep(2 * time.Minute)
```

Comment:
2 min wait feels long..

Reply:
This 2 wait is going to be removed before the PR gets merged to the master branch. The OpenAPI spec for EDRS was missing the task ID. This is a temporary workaround. Once the issue has been fixed we will be removing this wait.

Reply:
Done